doc/tor.md

```diff
@@ -29,9 +29,8 @@ outgoing connections, but more is possible.
     -onion=ip:port  Set the proxy server to use for Tor onion services. You do not
                     need to set this if it's the same as -proxy. You can use -onion=0
                     to explicitly disable access to onion services.
-                    Note: Only the -proxy option sets the proxy for DNS requests;
-                    with -onion they will not route over Tor, so use -proxy if you
-                    have privacy concerns.
+
+**Note:** _Only the -proxy option sets the proxy for DNS requests; with -onion they will not route over Tor, so use -proxy if you have privacy concerns._
```

Comment:
This change breaks up the formatting. I think it is better in the -onion section where it is currently and where it directly relates.

Reply:
Sorry I don't agree with this. I tried different things to highlight this part so that a user reading this doc for the first time do not miss this information. Or it will be helpful if you could suggest me a better way to highlight this.

Reply:
Sure. Can you keep the current position and *italicize* it to highlight it? This is a list of config options and it would be good to not break it up in the middle.

Reply:
> it will be helpful if you could suggest me a better way to highlight this.

Sure! Can you keep the current position and *italicize this part* or use **bold characters** to highlight it? This is a list of config options and it would be good to not break it up in the middle.

Reply:
I couldn't find a way to use italic or bold in a code block indented with four spaces.

Reply:
- 2 config options to set proxy
- Highlighted text about those options
- Other config options below

LGTM

![image](https://user-images.githubusercontent.com/13405205/117246703-b6143780-ae5a-11eb-8078-d45a866459a8.png)

Reply:
If you really want to pull out this text, why not place it at the top or the bottom of the list? Still, as this text refers to the `-onion` option, in my opinion it should remain there.

Reply:
Do you plan to address this?

Reply:
I am not sure if there is any other better way to highlight this. Top and Bottom will not be useful. A newbie or even a power user needs to read this while going to through proxy options. Highlighting DNS was one of the important parts of this PR (mentioned in description) as I have seen lot of people ignore it and DNS leaks affecting their privacy.

Reply:
I think it's in the right place already.

Reply:
This adds the bold formatting as @jonatack requested: https://github.com/harding/bitcoin/blob/095310798bc0622498515cf6d5c772a01c60c7c5/doc/tor.md

```diff
diff --git a/doc/tor.md b/doc/tor.md
index 2640a6109..31fa4f554 100644
--- a/doc/tor.md
+++ b/doc/tor.md
@@ -25,6 +25,7 @@ knows when upgrading to current and future Tor releases that support Tor v3 only
 The first step is running Bitcoin Core behind a Tor proxy. This will already anonymize all
 outgoing connections, but more is possible.
 
+<pre>
     -proxy=ip:port  Set the proxy server. If SOCKS5 is selected (default), this proxy
                     server will be used to try to reach .onion addresses as well.
                     You need to use -noonion or -onion=0 to explicitly disable
@@ -33,7 +34,7 @@ outgoing connections, but more is possible.
     -onion=ip:port  Set the proxy server to use for Tor onion services. You do not
                     need to set this if it's the same as -proxy. You can use -onion=0
                     to explicitly disable access to onion services.
-                    Note: Only the -proxy option sets the proxy for DNS requests;
+                    <b>Note:</b> Only the -proxy option sets the proxy for DNS requests;
                     with -onion they will not route over Tor, so use -proxy if you
                     have privacy concerns.
 
@@ -54,6 +55,7 @@ outgoing connections, but more is possible.
                     connections will be enabled when you use -proxy or -onion. Use
                     -noonion or -onion=0 if you want to be sure there are no outbound
                     onion connections over the default proxy or your defined -proxy.
+</pre>
 
 In a typical situation, this suffices to run behind a Tor proxy:
 
```

Reply:
Cool. I will make this change. I was trying bold and italic without `<pre></pre>` tags

Reply:
Implemented the suggestion in https://github.com/bitcoin/bitcoin/commit/a51c1b795544238f534c7273c9382357e44b05a6